ledger/ledger-api-test-tool/src/main/scala/com/daml/ledger/api/testtool/Cli.scala

```diff
@@ -197,6 +197,11 @@ object Cli {
       .action((_, _) => { println(BuildInfo.Version); sys.exit(0) })
       .text("Prints the version on stdout and exit.")
 
+    opt[Int]("ledger-clock-tick-interval")
+      .optional()
+      .action((interval, c) => c.copy(ledgerClockTickIntervalMs = interval))
+      .text("Specify the interval in ms at which the clock of the ledger under test ticks.")
```

Comment:
It's very difficult to understand the meaning of this flag from it's description and name and why one should choose change it. The name could probably be well suited for the context you have on the ledger you are testing. Please have a second look at the name and description of this option.

@rautenrieth-da Can you have a look at this one? You can probably suggest a more telling name for this.

Reply:
Perhaps a name like "ledger-clock-granularity"?

Reply:
Reading the code it looks like a timeout, rather than some form of ticks or granularity.

Reply:
it is basically an indicator of ledger clock granularity - the max time you will see between clock ticks on the ledger under test

Reply:
renamed and added better description